packages/liveblocks-core/src/lib/__tests__/position.test.ts

```diff
@@ -392,6 +393,9 @@ describe("makePosition", () => {
   test("before .1 lies .09", () =>
     expect(makePosition(undefined, ONE)).toBe(ZERO + NINE));
 
+  test("between .1 and .11 lies .105", () =>
+    expect(makePosition(ONE, asPos(ONE + ONE))).toBe(ONE + ZERO + MID));
```

Comment:
The bug is clearly visible here: this previously returned the equivalent of `.15` instead of `.105`!